mentat/parsing.py

```diff
@@ -132,7 +133,7 @@ def run_async_stream_and_parse_llm_response(
     except openai.error.InvalidRequestError as e:
         cprint(e, "red")
         exit()
-    except KeyboardInterrupt:
+    except UserInterrupt:
```

Comment:
Might want both `KeyboardInterrupt` and `UserInterrupt` here?

Probably at least `KeyboardInterrupt`, actually.

Reply:
Merp, this is overcomplicated. Simplified greatly in 110fffef49f1f74ad510d8eb84cbccf807fbae42. :)